.travis.yml

```diff
@@ -0,0 +1,22 @@
+matrix:
+  include:
+    - language: python
+      python:
+      - "3.6"
+      install:
+        - pip3 install -r requirements3.txt
+      script:
+        - make test
+        - make lint
+
```

Comment:
`make test` also runs the linter, so we don't need `make lint` here. We also have to add mypy checks:
https://github.com/lyft/amundsensearchlibrary/pull/4